apps/api/src/app/subscribers-v2/subscriber.controller.ts

```diff
@@ -28,7 +28,7 @@ export class SubscriberController {
         after: query.after,
         before: query.before,
         orderDirection: query.orderDirection || DirectionEnum.DESC,
-        orderBy: query.orderBy || 'createdAt',
+        orderBy: query.orderBy || '_id',
```

Comment:
same as i mentioned below 
sorting by `id` is an internal implementation detail.
we need to map `createdAt` to `id` internally

Reply:
IMO since it has its drawbacks we should keep it as `_id`. It's not correct to expose it as `createdAt`.